### DISCUSSION TITLE
[LIB] GameOfLife - Conway game

### DISCUSSION THREAD

**Discussion by AndyGFX** [paladin-t/bitty#33] [Libs And Tips]
[LIB] GameOfLife - Conway game

Download: [[LIB] GameOfLife](https://github.com/AndyGFX/AGFX_ProjectsData/blob/main/BittyEngine/LIB-GameOfLife/LIB-GameOfLife.zip)

![Screenshot](https://github.com/AndyGFX/AGFX_ProjectsData/blob/main/BittyEngine/LIB-GameOfLife/GameOfLife.png?raw=true)

**Comment by paladin-t** [comment]
The download and screenshots are unavailable, maybe it's linking to a private repo? You could just drag&drop .zip .png etc files into the edit area to re-upload.

**Comment by AndyGFX** [comment]
Hi,

sorry, yes repo was private. Now is public.
